Stack/js/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Create new 'ul' element
+ * @param {object} parent
+ * @returns {Element}
+ */
+function createUl(parent) {
+  let ul = document.createElement('ul');
+  let line = document.createElement('hr');
+
+  parent.appendChild(ul);
+  parent.appendChild(line);
+
+  return ul;
+}
+
+/**
+ * Show stack
+ * @param arr
+ */
+function stack(arr) {
+  const parent = document.getElementById('stack');
+  let ul = createUl(parent);
+  let count = arr.length;
+  let li;
+
+  for (let id = 0; id < count; id += 1) {
+    li = document.createElement('li');
+    li.innerHTML = arr.pop();
+    ul.appendChild(li);
+  }
+
+}
+
+/**
+ * Show queue
+ * @param arr
+ */
+function queue(arr) {
+  const parent = document.getElementById('queue');
+  let ul = createUl(parent);
+  let count = arr.length;
+  let li;
+
+  for (let id = 0; id < count; id += 1) {
```

Comment:
you can use simple version of array loop:

```
arr.forEach(() => {
  li = document.createElement('li');
  li.innerHTML = arr.shift();
  ul.appendChild(li);
});
```

How you can see, you don't need to setup loop and get count of array.  It's easier